Presentation/MainPDF/BackPageBtnView.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  PreviousButton.swift
+//  Reazy
+//
+//  Created by 김예림 on 1/25/25.
+//
+
+import SwiftUI
+
+struct BackPageBtnView: View {
+    
+    @EnvironmentObject private var viewModel: BackPageBtnViewModel
+    
+    var body: some View {
+        Button(action: {
+            viewModel.handleButnVisible()
+            viewModel.updateBackDestination()
+        }, label: {
```

Comment:
재사용 가능성이 있는 버튼 View 같은 경우에는 의존성을 주입하기 보단 클로저를 프로퍼티로 받는게 재사용으로 활용하기 더욱 좋아보입니다!!

```suggestion
    let action: () -> Void
    
    var body: some View {
        Button(action: {
            action()
        }, label: {
```

Reply:
오 이런 방식은 처음 알았어요 !! 짱짱